src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -168,6 +169,11 @@ def register_options(cls, register):
     register('--coverage-output-dir', metavar='<DIR>', default=None,
              help='Directory to emit coverage reports to. '
              'If not specified, a default within dist is used.')
+    register('--coverage-config-file', metavar='<FILE>', default=None,
```

Comment:
This should be part of the test run fingerprint since the coverage config can effect which code is instrumented and thus - potentially - different (even if that shouldn't ever affect a test result). So you'll want to pass `fingerprint=True`.